admin/card.py

```
@@ -0,0 +1,730 @@
+        # Left edge
+        points.extend([x1, y2 - radius, x1, y1 + radius])
+        # Top left corner
+        points.extend([x1, y1, x1 + radius, y1])
+        
+        return self.create_polygon(points, smooth=True, **kwargs)
+
+    def _on_click(self, event):
+        if self.command:
+            self.command()
+
+    def _on_hover(self, event):
```

Ruff (0.13.1)

Unused method argument: `event`

(ARG002)